.chloggen/elastic-exporter-version-rollback.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: roll back elasticsearch client to v7.17.7 due to incompatibility with older versions
```

Comment:
Do you mean `newer` version here?

And is there an upstream issue also tracking this?

Reply:
Goo point wording sound confusing. I meant older version of elaticsrarch. I will change it

Reply:
@MovieStoreGuy https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16310